raiwidgets/raiwidgets/_cohort.py

```diff
@@ -66,9 +79,20 @@ class CohortFilter:
                    will be applied.
     :type column: str
     """
+    PREDICTED_Y = 'Predicted Y'
+    TRUE_Y = 'True Y'
+    INDEX = 'Index'
+    CLASSIFICATION_OUTCOME = 'Classification Outcome'
```

Comment:
1.) it looks like the O should be lowercase based on what I see in error analysis code?  would be good to check.
2.) I think many of these are the same or similar to the constants in error analysis.  This really makes me wonder if something needs to be more "core" than error analysis, like a cohort package, and EA should take a dependency on this cohort package.

See:

https://github.com/microsoft/responsible-ai-toolbox/blob/main/erroranalysis/erroranalysis/_internal/cohort_filter.py#L14

Reply:
Thanks fixed lowercase 'o' for 1.

For 2, we should look at the contract between UI and SDK and see if we can leverage the code in _cohort.py. Probably outside of scope of this PR but I am open to looking for consolidating opportunities and creating a separate package for cohorts.